src/cmd/flux-account-service.py

```diff
@@ -202,6 +202,8 @@ def list_users(self, handle, watcher, msg, arg):
             handle.respond(msg, payload)
         except KeyError as exc:
             handle.respond_error(msg, 0, f"list-users: missing key in payload: {exc}")
+        except (ValueError, sqlite3.Error) as exc:
+            handle.respond_error(msg, 0, f"list-users: {exc}")
```

Comment:
Just checking on this commit. The commit message states:

> Condense the two except blocks to handle them on the same line instead of using two separate except blocks.

I'm probably missing something in the diff, but I just see this _additional_ except block added, but no `except` removed, so just making sure this does what the commit message says?

Reply:
Also, should the fall-through (`except Exception`) case also have the `list-user: ` prefix added?

Reply:
Oh duh, thank you for catching this 🤦 I got mixed up while editing one of these commits and never fixed the commit message. This should instead say something along the lines of "the `list-users` command was never handling either exception, but either could occur in the function...."

I'll edit this commit message.